daemon/indexing/listener/listener.js

```diff
@@ -431,6 +438,10 @@ const config = {
   // Verbose mode, includes dumping events on the console.
   verbose: args['--verbose'],
   // File to use for picking which block number to restart from
-  continueFile: args['--continue-file']
+  continueFile: args['--continue-file'],
+  // web3 provider url
+  web3Url: args['--web3-url'] || 'http://localhost:8545',
+  // ipfs url
+  ipfsUrl: args['--ipfs-url'] || 'http://origin-js:8080',
```

Comment:
I think `ipfsUrl` and `web3Url` would be better located in the `.env` file instead of as command line parameters.

Reply:
I think adding the ability to configure out of environment variables would be out of scope for this PR, but I'll add an issue for it.